### ISSUE TITLE
Analog output garbled

### ISSUE THREAD

**Issue by UndertakerInfinity** [MiSTer-devel/PSX_MiSTer#249]
I'm using an old NTSC CRT TV in composite through a VGA to NTSC active adapter.
Here's my relevant ini settings:
forced_scandoubler=1
ypbpr=0
composite_sync=0
vga_scaler=0

After a recent update (current year, sorry can't tell exactly which update it was) output started coming out garbled, like it does while MiSTer is starting up, but it stays like that.
Setting vga_scaler to 1 fixes the image, only now it's going through the scaler, which previously wasn't needed.

**Comment by sorgelig** [comment]
should be: `forced_scandoubler=0`

**Comment by birdybro** [comment]
https://mister-devel.github.io/MkDocs_MiSTer/advanced/crttable/ - Here's a reference table for all ways you can configure analog output that I'm aware of.

**Comment by UndertakerInfinity** [comment]
That reference table is great, I had not seen it before. However:
forced_scandoubler=0 showed no change. PSX core output is garbled, other cores work fine.
composite_sync=1 gave me test bars in the menu, couldn't get to the core.

**Comment by UndertakerInfinity** [comment]
did additional testing and setting forced_scandoubler=0 made every other core look like the PSX core.
setting forced_scandoubler=1 made every other core look good again, but not PSX.
Maybe PSX core is ignoring forced_scandoubler setting?
Ran update_all just in case, but no change on this issue.
Note I'm using a regular old CRT NTSC TV, not a PVM or a PC Monitor

**Comment by Kuba-J** [comment]
> did additional testing and setting forced_scandoubler=0 made every other core look like the PSX core. setting forced_scandoubler=1 made every other core look good again, but not PSX. Maybe PSX core is ignoring forced_scandoubler setting? Ran update_all just in case, but no change on this issue. Note I'm using a regular old CRT NTSC TV, not a PVM or a PC Monitor

There is no scandoubler in PSX core:

https://github.com/MiSTer-devel/PSX_MiSTer/issues/213

**Comment by birdybro** [comment]
Yup, this is the issue. You are not outputting to a 15kHz display, you are outputting to a 31kHz display, so you require the scandoubler, and the PSX core does not use that. You will require an external 15kHz --> 31kHz converter to make it work correctly or you can try your luck with super resolution video modes, but that's a lot of trial and error since compatibility depends on the target display.

**Comment by UndertakerInfinity** [comment]
oh vga scaler definitely works and it's what I'm doing right now. My TV has no resolution settings, it's an old component-only set. Will 480p mode or vga_scaler display interlaced content correctly? that's about the main reason to use a CRT for me, deinterlacing is not the same.

What's odd is that I had it without the scaler before that update (I default to no scaler and then add per-core exceptions: the menu and every arcade core that requires rotation).

**Comment by birdybro** [comment]
Ah, I think I know what might be happening. There are a new couple of options in the MiSTer.ini that you should take note of:

https://github.com/MiSTer-devel/Main_MiSTer/blob/master/MiSTer.ini#L5-L6

You might need to switch vga_mode= to ypbpr.

**Comment by UndertakerInfinity** [comment]
relevant things I forgot to share:

- video_mode=6
- TV doesn't have a resolution switch
- using vga output to an active converter to component. It handles PAL (looks a bit squished) and can position the image, so it's "smart".
- I intend to move to the new Y/C setting once I get the adapter, so the cores have more control over the signal

I tried vga_mode=ypbpr and got bluish image in every core, psx core still garbled. I think the converter is expecting standard vga.
I also tried vga_mode=cvbs with similar results (a dim, bluish, striped image).

Bottom line is:

- the settings shown in [#213](https://github.com/MiSTer-devel/PSX_MiSTer/issues/213) work right now.
- Before a recent update (December I guess?), it was working with vga_scaler=0

**Comment by UndertakerInfinity** [comment]
addendum: I ran a bunch of tests with vga_scaler=1, video_mode=6 and the PSX core's 240i to 480p hack.

hack off:
- non-interlaced content (tested on Wipeout) looks fine
- interlaced content (tested on Tekken 3) shows artifacts and low resolution as expected: MiSTer sends interlaced frames, the converter fits each of them to NTSC and displays them in full.

hack on:
- non-interlaced content still looks fine (no difference, good!)
- interlaced content looks smooth and high res. I guess the core rearranges the two interlaced frames into a single large 480p frame and sends it twice, so the converter picks the even/odd lines and forms the two frames with the right fields for NTSC? so I'm getting the actual interlaced image just as a PSX would show?

**Comment by birdybro** [comment]
@UndertakerInfinity Is this resolved?
